README.md

```diff
@@ -1,49 +1,55 @@
-Make sure all tests are passing when creating a pull request,
+### Infrastructure (IDE, Version Control, Issue Tracking, ...)
+[![PhpStorm](https://img.shields.io/badge/PhpStorm-recommended-green?labelColor=purple)](https://www.jetbrains.com/phpstorm/)
+[![Symfony](https://img.shields.io/badge/Symfony-purple)](https://symfony.com/)
+[![Git](https://img.shields.io/badge/Git-purple)](https://git-scm.com/)
+[![Jira](https://img.shields.io/badge/Jira-purple)](https://jira.catrob.at/secure/Dashboard.jspa)
+[![Docker](https://img.shields.io/badge/Docker-purple)](https://www.docker.com/)
 
-[![Docker](https://img.shields.io/badge/Docker-supported-brightgreen.svg)](https://github.com/Catrobat/Catroweb/wiki/Docker)
+### Knowledge Repository
+[![Confluence](https://img.shields.io/badge/Confluence-orange)](https://confluence.catrob.at/)
 
-##### Tools integrated in the project:
+### Communication
+[![Slack](https://img.shields.io/badge/Slack-green)](https://slack.com/)
+[![Discord](https://img.shields.io/badge/Discord-green)](https://discord.com/)
 
-[![Symfony](https://img.shields.io/badge/Framework-Symfony-0379B8.svg)](https://symfony.com/)
-[![Openapi-generator](https://img.shields.io/badge/API-openapi_generator-0379B8.svg)](https://github.com/OpenAPITools/openapi-generator)
-[![Elastic](https://img.shields.io/badge/Search-Elastic-FF6940.svg)](https://www.elastic.co/)
-[![Php-cs-fixer](https://img.shields.io/badge/Php_Code_Style-Php_CS_Fixer-informational.svg)](https://github.com/FriendsOfPHP/PHP-CS-Fixer)
-[![PhpStan](https://img.shields.io/badge/Php_Code_Quality-PhpStan,_Psalm,_PhpDD,_PhpCPD_-informational.svg)](https://github.com/phpstan/phpstan)
-[![Webpack Encore](https://img.shields.io/badge/Asset_Bundler-Webpack_Encore-75AFCC.svg)](https://github.com/symfony/webpack-encore)
-[![Npm standard](https://img.shields.io/badge/JS_Code_Style-standard-green.svg)](https://standardjs.com)
-[![ESLint](https://img.shields.io/badge/JS_Code_Quality-ESLint-green.svg)](https://eslint.org/)
-[![Sass](https://img.shields.io/badge/CSS_Preprocessor-Dart_Sass-ff69b4.svg)](https://sass-lang.com/)
-[![StyleLint](https://img.shields.io/badge/CSS_Code_Style-StyleLint-ff69b4.svg)](https://stylelint.io/)
 
-**Have fun!**
+## Start Contribution
+If you feel confident that your tech stack matches our requirements, then we invite you to [start contributing](https://github.com/Catrobat/Catroweb/blob/develop/.github/onboarding.md)!
 
-# License #
-[License](https://catrob.at/licenses) of our project (mainly AGPL v3).
+## Issues
+You found a bug? 
+1. Perfect, please check the [open issues on Jira](https://jira.catrob.at/browse/SHARE-529?jql=project%20%3D%20%22SHARE%22%20AND%20issuetype%20%3D%20Bug%20AND%20resolution%20%3D%20Unresolved%20ORDER%20BY%20priority%20DESC%2C%20key%20DESC) first to avoid duplicate tickets!
```

Comment:
Correct Jira URL: https://jira.catrob.at/projects/SHARE/issues/?filter=allopenissues
```suggestion
1. Perfect, please check the [open issues on Jira](https://jira.catrob.at/projects/SHARE/issues/?filter=allopenissues) first to avoid duplicate tickets!
```